thrift.js

```diff
@@ -177,13 +177,13 @@ Thrift.prototype._asyncParse = function _asyncParse(filename, allowIncludeAlias,
         return model._asyncParseIncludedModules(filename, allowIncludeAlias, cb);
     }
 
-    model.fs.readFile(filename, function (err, source) {
+    model.fs.readFile(filename, 'utf-8', function (err, source) {
```

Comment:
The `model.fs.readFile` can be a custom function, can we update the documentation in the README to inform users that they need to accept three arguments (`Non filesystem and asynchronous source loading`)?